code/game/gamemodes/traitor/traitor.dm

```diff
@@ -345,6 +345,34 @@
 	if(M && M != traitor_mob)

```

Comment:
Perhaps change this, so it runs in all game modes.
Then people can't use this to metagame the round type.